mappings/net/minecraft/server/command/RideCommand.mapping

```diff
@@ -0,0 +1,23 @@
+CLASS net/minecraft/unmapped/C_twdmsalp net/minecraft/server/command/RideCommand
+	FIELD f_cqvivuod FAILURE_GENERIC_EXCEPTION Lcom/mojang/brigadier/exceptions/Dynamic2CommandExceptionType;
+	FIELD f_ibawxhsz ALREADY_RIDING_EXCEPTION Lcom/mojang/brigadier/exceptions/Dynamic2CommandExceptionType;
+	FIELD f_ybzixihk NOT_RIDING_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
+	FIELD f_ylstdqza MOUNT_FAILURE_CANT_RIDE_PLAYERS_EXCEPTION Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
```

Comment:
```suggestion
	FIELD f_ylstdqza MOUNT_FAILURE_CANNOT_RIDE_PLAYERS_EXCEPTION Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
```

I prefer to avoid the `can't` abbreviation, the proper form is only two letters more

Reply:
Yep, that's better.